src/main/python/deep_qa/layers/backend/repeat.py

```diff
@@ -0,0 +1,39 @@
+from keras import backend as K
+from keras.layers import Layer
+from overrides import overrides
+
+
+class Repeat(Layer):
+    """
+    This `Layer` calls `K.repeat_elements` on both the input and the mask, after calling
+    `K.expand_dims`.
+
+    If the mask is not `None`, we must be able to call `K.expand_dims` using the same axis
+    parameter as we do for the input.
+
+    Input:
+        A tensor of arbitrary shape.
```

Comment:
Maybe add bullets for each item? http://docutils.sourceforge.net/docs/user/rst/quickref.html#bullet-lists